tests/staticfiles_tests/project/documents/cached/styles.css

```diff
@@ -1 +1,19 @@
 @import url("other.css");
+
```

Comment:
I would added commented URLs to the `ignored.css` and new tests cases to the existing `test_path_ignored_completely()`.